combat-trainer.lic

```diff
@@ -2854,8 +2858,8 @@ class TrainerProcess
 
     game_state.sheath_whirlwind_offhand if game_state.currently_whirlwinding
     DRC.retreat
-    fput('engage') unless game_state.npcs.empty? || game_state.retreating?
-    DRC.bput("get my #{@almanac}", 'You get', 'What were')
+    game_state.engage unless game_state.npcs.empty? || game_state.retreating?
+    DRCI.get_item?(@almanac)
```

Comment:
```suggestion
    DRC.bput("get my #{@almanac}", 'You get', 'What were')
```

Reply:
I agree it should be changed, just not in this PR.

Reply:
copy.